app/models/course.rb

```diff
@@ -183,7 +183,7 @@ def new_draft_attributes
 
   scope :with_recruitment_cycle, ->(year) { joins(provider: :recruitment_cycle).where(recruitment_cycle: { year: year }) }
   scope :findable, -> { joins(:site_statuses).merge(SiteStatus.findable) }
-  scope :with_vacancies, -> { joins(:site_statuses).merge(SiteStatus.with_vacancies) }
+  scope :with_vacancies, -> { joins(:site_statuses).merge(SiteStatus.findable.with_vacancies) }
```

Comment:
was this meant to be changed?

given 

```ruby
# app/models/site_status.rb 
scope :with_vacancies, -> { where.not(vac_status: :no_vacancies).findable }
```

Reply:
You're right, it shouldn't need to be changed seeing as site status already covers it